backend/src/main/java/gov/cdc/usds/simplereport/db/repository/TestEventRepository.java

```diff
@@ -18,7 +21,19 @@
 
 	public TestEvent findFirst1ByPatientOrderByCreatedAtDesc(Person p);
 
+	public TestEvent findFirst1ByOrganizationAndInternalId(Organization o, UUID id);
+
 	// Need to control how this query is built. "between" is too vague.
 	@Query("FROM #{#entityName} q WHERE q.createdAt > :before AND q.createdAt <= :after ORDER BY q.createdAt DESC ")
 	public List<TestEvent> queryMatchAllBetweenDates(Date before, Date after);
+
+	@Query( value = "SELECT * FROM " +
+			"    (SELECT MAX(created_at) each_created_at " +
+			"        FROM simple_report.test_event inner_evt " +
+			"        WHERE inner_evt.facility_id = :facility_id " +
+			"     GROUP BY test_order_id) AS latest_events " +
+			"INNER JOIN simple_report.test_event events " +
+			"ON events.created_at = latest_events.each_created_at " +
+			"ORDER BY created_at", nativeQuery = true)
+	public List<TestEvent> getTestEventResults(@Param("facility_id")UUID facility_id);
```

Comment:
1) You don't need the `@Param` annotation, and you should use camelCase for method parameters.
2) This doesn't seem like the right query, though it'll probably work in many cases before it starts mysteriously failing—we should have `test_order_id` in the join.
3) This is more simply written 

```sql
SELECT DISTINCT ON (test_order_id) *
FROM test_event
order by test_order_id, created_at desc
```